templates/test/overview.html.ep

```diff
@@ -83,4 +83,80 @@
             </tbody>
         </table>
     % }
+    <div class="panel panel-default">
+            <div class="panel-heading">Filter</div>
+            <div class="panel-body">
+                <form action="#" type="get" id="filter-form">
+                    <div class="form-group">
+                        <strong>Result</strong>
+                        <label><input value="passed" name="result" type="checkbox" id="filter-passed"> Passed</label>
+                        <label><input value="incomplete" name="result" type="checkbox" id="filter-incomplete"> Incomplete</label>
+                        <label><input value="softfailed" name="result" type="checkbox" id="filter-softfailed"> Softfailed</label>
+                        <label><input value="failed" name="result" type="checkbox" id="filter-failed"> Failed</label>
+                        <label><input value="running" name="result" type="checkbox" id="filter-running"> Running</label>
+                    </div>
+                    <div class="form-group">
+                        <strong>Architecture</strong>
+                        <input type="text" class="form-control" name="arch" placeholder="Architecture" id="filter-arch">
+                    </div>
+                    <div class="form-group">
+                        <strong>Misc</strong>
+                        <label><input value="1" name="todo" type="checkbox" id="filter-todo"> TODO</label>
+                    </div>
+                    <button type="submit" class="btn btn-default">Apply</button>
+                </form>
+            </div>
+    </div>
 </div>
+<style type="text/css">
+#filter-form strong {
+    display: block;
+}
+#filter-form .form-group label {
+    margin-left: 0px;
+    padding-left: 0px;
+    margin-right: 20px;
+    font-weight: normal;
+}
+#filter-form .form-group input {
+    margin-right: 2px;
+    padding-right: 0px;
+}
+</style>
+<script type="text/javascript">
+// ensure TODO is false by default because the Browser auto remembers previous state
```

Comment:
no script and no style in templates please. They are not minified and not cached this way